src/Site/wwwroot/favicons/browserconfig.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+  <msapplication>
+    <tile>
+      <square70x70logo src="/favicons/mstile-70x70.png"/>
+      <square150x150logo src="/favicons/mstile-150x150.png"/>
+      <square310x310logo src="/favicons/mstile-310x310.png"/>
+      <wide310x150logo src="/favicons/mstile-310x150.png"/>

```

Comment:
Similar to the comment above, could we remove the `/` or `/favicons/` at the beginning of each src attribute? I'm not if it'll use the path relative to the `browserconfig.xml` file, or the path relative the the application.

If we moved it to the app root, then it's easy... :smiley: